lambda-notes/src/components/TopicMap.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import MiniMap from './MiniMap';
+
+const dotsArr = [{number: 'one', dotID: Math.floor(Math.random() * 1000000), bgColor: '', lit: false, loaded: false},
+              {number: 'two', dotID: Math.floor(Math.random() * 1000000), bgColor: '', lit: false, loaded: false},
```

Comment:
interesting article on Math.random in Javascript having problems when used for UUIDs at large scale https://medium.com/@betable/tifu-by-using-math-random-f1c308c4fd9d

Obviously doesn't matter for a small number of notes though, just food for thought